system/screen.c

```diff
@@ -10,8 +10,20 @@
 #include "font.h"
 #include "vmem.h"
 
+#include "string.h"
+
 #include "screen.h"
 
+//------------------------------------------------------------------------------
+// Private Types
+//------------------------------------------------------------------------------
+
+typedef enum  __attribute__ ((packed)) {
```

Comment:
``-fshort-enums``? The ``packed`` attribute is quiet unusual for anything except structures.

Reply:
-fshort-enums would affect all enums, which we don't necessarily want.

Reply:
This is true but it shouldn't break anything for bare-metal code.
OK, I got it, I should check it myself first.